pacai/util/probability.py

```diff
@@ -68,8 +64,8 @@ def sample(distribution, values = None):
 
     return values[i]
 
-def sampleFromCounter(ctr):
-    items = sorted(list(ctr.items()))
+def sampleFromDict(dictDistribution):
```

Comment:
Doesn't `sample` have this functionality?

Reply:
Yes, I'm deleting this method.